varia/sass/components/footer/_config.scss

```diff
@@ -12,6 +12,6 @@ $config-footer: (
 			"css-var": map-deep-get($config-global, "font", "family", "primary", "css-var"),
 		),
 		"size": map-deep-get($config-global, "font", "size", "sm"),
-		"line-height": map-deep-get($config-global, "font", "line-height", "sm"),
```

Comment:
`$config-global` doesn't have this value, so I've changed it to `body` instead. Is that ok in this instance?